nautobot/dcim/migrations/0046_increase_device_asset_tag_size_limit.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.20 on 2023-07-21 16:07
```

Comment:
We can't renumber any of the migrations that already exist in 1.6.0 because that'll cause problems on upgrade, as I understand it. Instead these migrations should keep the numbers that exist in 1.6 (0020 through 0023) and all of the new 2.0 migrations need to be incremented by 4 to make space.